helm/istio/charts/istio_cni/templates/daemonset.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     heritage: {{ .Release.Service }}
-    maistra-version: 0.12.0
```

Comment:
We auto-generate the version that the rest of Istio in the build process. Would it make sense to update these there as well?

Reply:
Yes, I think these should be removed.  See @knrc's comment below.

Reply:
Ah ok. I misunderstood that one as talking about the helm labels.

Reply:
The script does not currently modify this version, I'll push an update which handles this.

Reply:
I looked deeper into this and the script does handle it, the original overlay therefore resulted in two maistra-version labels being included within the daemonset